docs/assets/lifecycle.puml

```diff
@@ -0,0 +1,131 @@
+@startuml
```

Comment:
Please keep the original file, and name it like lifecycle_cn.puml, lifecycle_en.puml